connector/src/main/java/org/jboss/as/connector/subsystems/jca/JcaExtension.java

```diff
@@ -80,7 +80,7 @@ public class JcaExtension implements Extension {
 
     public static final String SUBSYSTEM_NAME = "jca";
 
-    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(5, 0, 0);
+    private static final ModelVersion CURRENT_MODEL_VERSION = ModelVersion.create(5, 1, 0);
```

Comment:
Please use 6.0 for this and the XSD. We try to reserve minor versions for use downstream in case that's needed. The version numbers don't really have any semantic meaning.

Reply:
fixed

Reply:
@tadamski This is still 5.1.

https://github.com/tadamski/wildfly/pull/4 changes that to 6.0 and also adds a mixed domain transformer and parser handling for old namespaces such that they produce the same ops that they did before (matching what the xsd says they'll do.)

Reply:
oops, sorry, fixed